### DISCUSSION TITLE
New ComfyUI Logo concept

### DISCUSSION THREAD

**Discussion by aimpowerment** [Comfy-Org/ComfyUI#3049] [Ideas]
Hello Folks,
ive create some logo concepts for ComfyUI. It´s funny to see when you turn the C with a node symbol, you can also read UI.
Hope you like it and we use it in future
Best wishes 
Alberto

![g81](https://github.com/comfyanonymous/ComfyUI/assets/163556314/7d483ea4-2167-47cd-a6ec-242cbcc77288)
![g61](https://github.com/comfyanonymous/ComfyUI/assets/163556314/201ad5de-2896-43cd-b089-e4da7d9da91f)

**Comment by ICHarmony** [comment]
image generated by FLUX-Schnell Apache 2.0 licensed model and inspired by the logo of browser icon
![ComfyUI_00003_](https://github.com/user-attachments/assets/2bb21a93-e724-414f-b1e9-e49dcaf21fa5)

**Comment by ICHarmony** [comment]
FLUX-dev for personal use only I believe
![ComfyUI_00002_](https://github.com/user-attachments/assets/3fb9eb30-5220-4308-a981-99555a50d212)
